clients/python-wrapper/pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[build-system]
+requires = ["setuptools>=77"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lakefs"
+version = "0.14.0"
+description = "lakeFS Python SDK Wrapper"
+keywords = ["OpenAPI", "OpenAPI-Generator", "lakeFS API", "Python Wrapper"]
+requires-python = ">=3.9"
+readme = "README.md"
+license = "Apache-2.0"
+authors = [
+    { name = "Treeverse", email = "services@treeverse.io" },
+]
+dependencies = [
+    "lakefs-sdk>=1.50,< 2",
+    "pyyaml~=6.0.1",
+]
+classifiers = [
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: Software Development :: Version Control",
+    "Topic :: Utilities",
+    "Typing :: Typed",
+]
+
+[dependency-groups]
+all = [ "boto3 >= 1.26.0" ]
+aws-iam = [ "boto3 >= 1.26.0" ]
```

Comment:
hi, `dependency-groups` and `extras` (aka `project.optional-dependencies`) aren't the same thing. `extras` are for package distributions whereas `dependency-groups` are project dependencies, similar to `requirements.txt` files.

Reply:
got it, so `test` moves into a dependency group, while `all` and `aws-iam` return to `optional-dependencies`?